src/postgraphql/http/createPostGraphQLHttpRequestHandler.js

```diff
@@ -21,14 +23,26 @@ const httpError = require('http-errors')
 const parseUrl = require('parseurl')
 const finalHandler = require('finalhandler')
 const bodyParser = require('body-parser')
+const sendFile = require('send')
+
+const { POSTGRAPHQL_ENV } = process.env
 
 const debugGraphql = new Debugger('postgraphql:graphql')
 const debugRequest = new Debugger('postgraphql:request')
 
+const graphiqlDirectory = resolvePath(__dirname, '../graphiql/public')
+
 const favicon = new Promise((resolve, reject) => {
   readFile(resolvePath(__dirname, '../../../resources/favicon.ico'), (error, data) => {
     if (error) reject(error)
-    else resolve(data)
+    else resolve(data.toString())
+  })
+})
+
+const origGraphiqlHtml = new Promise((resolve, reject) => {
+  readFile(resolvePath(__dirname, '../graphiql/public/index.html'), (error, data) => {
+    if (error) reject(error)
+    else resolve(data.toString())
```

Comment:
It's more efficient to pass `{encoding: 'utf8'}` to `readFile` (because then it doesn't have to allocate a buffer and then immediately turn it into a string); also it'd be more convenient to use a promise-backed `readFile`; e.g. 

```
import {readFile} from 'fs-promise';
...
const origGraphiqlHtml = readFile(resolvePath(...), {encoding: 'utf8'})
```